CMakeLists.txt

```diff
@@ -232,10 +232,12 @@ if (BUILD_SGAME)
 
     if (BUILD_GAME_NACL AND NOT (FORK EQUAL 2))
         include(ExternalProject)
-        ExternalProject_Add_Step(nacl-vms cbse-trigger
-                                 COMMENT "Finished CBSE generation, ready to build NaCl"
-                                 DEPENDS ${sgame_GENERATED_CBSE}
-                                 DEPENDERS build)
+        foreach(NACL_VMS_PROJECT ${NACL_VMS_PROJECTS})
+            ExternalProject_Add_Step(${NACL_VMS_PROJECT} cbse-trigger
+                COMMENT "Finished CBSE generation, ready to build NaCl"
```

Comment:
How about adding the project name in the comment so the message isn't identical every time?

Reply:
Done.